### PR TITLE
SC: Extend code coverage of BridgeManager

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes
This PR extends code coverage of BridgeManager.
The covered code is to handle request value transfer events.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: node/sc/bridge_manager_test.go [hunk 1]
```diff
@@ -201,42 +207,10 @@ func TestBridgeManager(t *testing.T) {
...
-						&bind.TransactOpts{From: cAuth.From, Signer: cAuth.Signer, GasLimit: testGasLimit},
-						ev.Raw.TxHash, ev.From, ev.To, nftAddr, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.Uri, ev.ExtraData)
-					if err != nil {
-						log.Fatalf("Failed to HandleERC721Transfer: %v", err)
-					}
-					fmt.Println("HandleERC721Transfer Transaction by event ", tx.Hash().Hex())
-					sim.Commit() // block
-				}
+				bridgeInfo.AddRequestValueTransferEvents([]*RequestValueTransferEvent{ev})
+				bridgeInfo.processingPendingRequestEvents()
```
Comment (ehnuje) [inline node/sc/bridge_manager_test.go]: Could you please explain this code change with detail?

Comment (KimKyungup) [reply node/sc/bridge_manager_test.go]: I added some comments. :)
1st routine is to insert the event to bridgeInfo's event list.
2nd routine is to handle the event in the event list.

File: node/sc/bridge_manager_test.go [hunk 2]
```diff
@@ -488,32 +469,13 @@ func TestBridgeManagerWithFee(t *testing.T) {
...
-					fmt.Println("HandleERC20Transfer Transaction by event ", tx.Hash().Hex())
-					sim.Commit() // block
-				}
+				// insert the value transfer request event to the bridge info's event list.
+				pBridgeInfo.AddRequestValueTransferEvents([]*RequestValueTransferEvent{ev})
+
+				// handle the value transfer request event in the event list.
+				pBridgeInfo.processingPendingRequestEvents()
 
+				sim.Commit() // block
```
Comment (jeonghonah) [inline node/sc/bridge_manager_test.go]: Question: what is meaning of the `// block` ?

File: node/sc/bridge_manager_test.go [hunk 3]
```diff
@@ -152,6 +154,10 @@ func TestBridgeManager(t *testing.T) {
 	assert.NoError(t, err)
 	sim.Commit() // block
 
+	// Register tokens on the bridgeInfo
+	bridgeInfo.RegisterToken(tokenAddr, tokenAddr)
```
Comment (jeonghonah) [inline node/sc/bridge_manager_test.go]: The name of the BridgeInfo's `RegisterToken()` is confusing. How about adding `bridge.RegisterToken()` inside the method in future PR?

File: node/sc/bridge_manager_test.go (hunk 2 above)
Comment (KimKyungup) [reply node/sc/bridge_manager_test.go]: It means mining block. :)

File: node/sc/bridge_manager_test.go (hunk 3 above)
Comment (KimKyungup) [reply node/sc/bridge_manager_test.go]: It's good. I will refactor the below code as you commented.
```
func (sbapi *SubBridgeAPI) RegisterToken(cBridgeAddr, pBridgeAddr, cTokenAddr, pTokenAddr common.Address) error {
	if !sbapi.sc.bridgeManager.IsValidBridgePair(cBridgeAddr, pBridgeAddr) {
		return ErrInvalidBridgePair
	}

	cBi, cExist := sbapi.sc.bridgeManager.GetBridgeInfo(cBridgeAddr)
	pBi, pExist := sbapi.sc.bridgeManager.GetBridgeInfo(pBridgeAddr)

	if !cExist || !pExist {
		return errors.New("bridge does not exist")
	}

	err := cBi.RegisterToken(cTokenAddr, pTokenAddr)
	if err != nil {
		return err
	}

	err = pBi.RegisterToken(pTokenAddr, cTokenAddr)
	if err != nil {
		return err
	}

	cBi.account.Lock()
	tx, err := cBi.bridge.RegisterToken(cBi.account.GetTransactOpts(), cTokenAddr, pTokenAddr)
	if err != nil {
		cBi.account.UnLock()
		return err
	}
	cBi.account.IncNonce()
	cBi.account.UnLock()
	logger.Debug("scBridge registered token", "txHash", tx.Hash().String(), "scToken", cTokenAddr.String(), "mcToken", pTokenAddr.String())

	pBi.account.Lock()
	tx, err = pBi.bridge.RegisterToken(pBi.account.GetTransactOpts(), pTokenAddr, cTokenAddr)
	if err != nil {
		pBi.account.UnLock()
		return err
	}
	pBi.account.IncNonce()
	pBi.account.UnLock()
	logger.Debug("mcBridge registered token", "txHash", tx.Hash().String(), "scToken", cTokenAddr.String(), "mcToken", pTokenAddr.String())

	logger.Info("Register token", "scToken", cTokenAddr.String(), "mcToken", pTokenAddr.String())
	return nil
}
```

**Pull request merged**